content/docs/arvo/clay/marks/examples.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "Examples"
+weight = 4
+template = "doc.html"
++++
+
+These are the files used in the [Writing Marks](/docs/arvo/clay/marks/writing-marks) guide.
+
+## `/lib/csv/hoon`
```

Comment:
There's already a `/lib/csv/hoon` https://github.com/urbit/urbit/blob/master/pkg/arvo/lib/csv.hoon so this is probably going to confuse the reader without some clarification

Reply:
dammit I never noticed that existed. I'll change the name of the library and add a clarifying note